docs/guides/1-step-by-step-guide.md

```diff
@@ -133,10 +133,10 @@ import { RouteRenderer } from 'yarr';
 <RouteRenderer
   pendingIndicator={<p>...pending loading </p>}
   routeWrapper={({ Route }) => (
-     <>
+     <div>
```

Comment:
Using `div` here to keep the sandbox light and not require `jsx` options in tsconfig and avoid adding explanation for that as it is beyond scope.

Reply:
We should allow anyone to write valid react code straight from our sandbox, so let's take the extra step to configure those options.

Reply:
Sounds good!